composer.json

```diff
@@ -67,7 +67,8 @@
     },
     "config": {
         "preferred-install": "dist",
-        "sort-packages": true
+        "sort-packages": true,
+        "process-timeout": 600
```

Comment:
Why this?

Tip: when you make changes like this, that are likely not going to be immediately apparent to others, please add an explanation in the description of your PR.

Reply:
increased timeout because my machine was getting composer timeout errors when running tests... I own macbook pro 2016, it's not the slowest computer but certain tests were timing out. ( default timeout is 300, so I don't think this needs much explanation)

Reply:
Fair.

> default timeout is 300, so I don't think this needs much explanation

Sometimes the most innocuous changes can have significant effects.🙂